apps/backend/prisma/migrations/20260625120000_publishing_workflow/migration.sql

```
@@ -0,0 +1,91 @@
+  "deletedAt" IS NULL
+  AND "publiclyVisible" = true
+  AND (
+    "majorEventId" IS NULL
+    OR "majorEventId" IN (
+      SELECT "id"
+      FROM "major_events"
+      WHERE "deletedAt" IS NULL
+    )
+  );
+
+CREATE INDEX "events_publicationState_idx" ON "events"("publicationState");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)